src/metabase/sync/sync_metadata/fks.clj

```diff
@@ -51,7 +51,10 @@
                       ;; - fk_target_field_id is NULL and the new target is not NULL
                       ;; - fk_target_field_id is not NULL but the new target is different and not NULL
                       [pk-field-id-query :pk]
-                      [:= :f.fk_target_field_id nil]]
+                      [:and
```

Comment:
Is this :and required? Seems weird

Reply:
definitely not, thanks 9472594a89aadfaa1c58abb2dd276674bfc9192a